Samples/2a-arm-validation/shell/stderr.txt

```diff
@@ -6,6 +6,8 @@ ERROR (ListByOperationsValidation/M1003/SDKViolation): Operation Usage_List must
         Path: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-storage/2015-06-15/swagger/storage.json#$.paths["/subscriptions/{subscriptionId}/providers/Microsoft.Storage/usages"].get.operationId
 ERROR (OperationsAPIImplementationValidation/M3023/RPCViolation): Operations API must be implemented for '/providers/Microsoft.Storage/operations'.
         Path: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-storage/2015-06-15/swagger/storage.json#$.paths
+ERROR (PutRequestResponseValidation/M2017/SDKViolation): "A PUT operation request body schema must be the same as its 200 response schema. Operation: StorageAccounts_Create"
```

Comment:
Just a thought, scrolling through these messages: Maybe it makes sense to write what the schemas involved are (something like `A PUT operation request body schema (found 'ABC') must be the same as its 200 response schema (found 'XYZ'). Operation: StorageAccounts_Create`).
That way we know what we're dealing with even without looking at the Swagger :-)